frontend/public/module/k8s/types.ts

```diff
@@ -326,6 +326,20 @@ export type DeploymentKind = {
   };
 } & K8sResourceCommon;
 
+export type AppliedClusterResourceQuotaKind = {
+  spec?: {
+    selector?: MatchLabels | Selector;
+    quota?: { hard: { [key: string]: number }; scopes?: string[] };
```

Comment:
`quota` is missing `ScopeSelector` field, check
https://github.com/openshift/api/blob/be1be0e89115702f8b508d351c4f5c9a16e5ae95/quota/v1/types.go#L37
and 
https://github.com/openshift/api/blob/be1be0e89115702f8b508d351c4f5c9a16e5ae95/vendor/k8s.io/api/core/v1/types.go#L5749